exercises/all-your-base/all-your-base.sml

```diff
@@ -0,0 +1,2 @@
+fun rebase (fromBase: int) (xs: int list) (toBase: int): int list option =
+  raise Fail "'rebase' has not been implemented"
```

Comment:
That will teach me not to use the generator :) I'll update this to match the generator output. I don't have strong feelings one way or the other about `"{function} is not implemented"` vs `"{function} has not been implemented"`